### ISSUE TITLE
How to convert U256 to stringified u64?

### ISSUE THREAD

**Issue by SeunLanLege** [tomusdrw/rust-web3#84]
hey guys, probably a noob question. but i'd like to convert the `U256` gotten from the `balance` function to a human readable number. I assume a string u64 is the only way to do this?

i tried using the `low_u64` but i'm not even sure what that does, but the number it returns is wrong. 

code 👇 

```rust
extern crate web3;
use web3::*;
use types::{U256, Address};
use futures::{Future};

#[derive(Debug)]
struct Mapping {
	address: Address,
	balance: u64
}

impl Mapping {
	fn new (address: Address, balance: U256) -> Mapping {
		Mapping {
			address,
			balance: balance.low_u64()
		}
	}
}

fn main() {
	let (_eloop, transport) = web3::transports::Http::new("http://localhost:8545").unwrap();
	let web3 = web3::Web3::new(transport);
	let accounts = web3.eth().accounts().wait().unwrap();

	let balances: Vec<Mapping> = accounts.into_iter().fold(Vec::new(),|mut accum, address| {
		let balance = web3.eth().balance(address, None).wait();
		accum.push(Mapping::new(address, balance.unwrap()));
		accum
	});

	for balance in balances {
		println!("Balance : {:?}", balance)
	}
}
```

```bash
ganache-cli -e 30 -a 1
Ganache CLI v6.0.3 (ganache-core: 2.0.2)

Available Accounts
==================
(0) 0x6ce23c45eb5b818fdaf76944ba5dd5cf56990a83

Private Keys
==================
(0) 12059b88a7fd4db2e973e4c9670d344b119cd7f9b7f21324bf24245beb9eb4fc

HD Wallet
==================
Mnemonic:      image want animal chimney know dune age board world volcano great subway
Base HD Path:  m/44'/60'/0'/0/{account_index}

Listening on localhost:8545
eth_accounts
eth_getBalance

```

```bash
cargo run                                                                                                                                                                                    master ✚ ◼
   Compiling cryptobot v0.1.0 (file:///home/seunlanlege/Projects/cryptobot)
    Finished dev [unoptimized + debuginfo] target(s) in 2.97 secs
     Running `target/debug/cryptobot`
Balance : Mapping { address: 0x6ce23c45eb5b818fdaf76944ba5dd5cf56990a83, balance: 7766279631452241920 }



```

**Comment by SeunLanLege** [comment]
@tomusdrw please help 😢

**Comment by aleksuss** [comment]
@SeunLanLege I use this func 
```rust
fn convert_u256_to_u64(input: U256) -> Option<u64> {
    Some(BigEndian::read_u64(&(*input)[24..32]))
}
```

**Comment by SeunLanLege** [comment]
@aleksuss it doesn't work, is there now way to do basic arithmetic with `U256` ? like add to and subtract from and convert balance from `wei` to `ether`. I would really like to use rust in my project, but a simple operation is taking to long to achieve. I've already written an implementation of my project in js, and if i can't get this lib to work, i'll just use the js implementation.

**Comment by tomusdrw** [comment]
@SeunLanLege Return types are currently just wrappers on 32 bytes vectors. Integration of proper fixed-size number library is pending (see #76).
For now you need to convert returned values to https://crates.io/crates/ethereum-types or https://crates.io/crates/ethcore-bigint
